sandbox/Test.scala

```diff
@@ -1,5 +1,5 @@
 object Test {
   def main(args: Array[String]): Unit = {
-    println("Hello, World!")
+    println("Hello, world!")
```

Comment:
This might be changing the originally hello world from Scala Days and the first release.

Reply:
Yeah, will revert this.